src/Simulation/Simulators.Tests/Circuits/ResourcesEstimator.qs

```diff
@@ -48,6 +48,21 @@ namespace Microsoft.Quantum.Simulation.Simulators.Tests
             T(q[0]);
         }
     }
+
+    operation QubitReuseWithOptimizedDepth() : Unit
```

Comment:
It would be good to add the case filed in the issue that spawned this work.

Reply:
QubitReuseSequentialTest and QubitReuseSimultaneousTest are the tests for the specific customer issue.